src/plugins/WebOfTrust/util/DelayedBackgroundJob.java

```diff
@@ -0,0 +1,26 @@
+package plugins.WebOfTrust.util;
+
+/**
+ * A {@link BackgroundJob} that is executed after triggers have been aggregated for some delay.
+ *
+ * @author bertm
+ */
+public interface DelayedBackgroundJob extends BackgroundJob {
+    /**
+     * Triggers the execution of the job after either aggregating triggers for at most some delay
+     * specified by the implementation or as soon as the currently running job finishes, whichever
+     * comes last. Implementations must ensure that this method is safe to invoke from any thread
+     * at any moment.
+     */
+    @Override
+    public void triggerExecution();
```

Comment:
Thanks for trying to explain this in depth.
I accept that it can mathematically defined that way, but phrasing the JavaDoc like that doesn't properly make the intention why the jobs are used in WOT clear: To delay execution for *at least* "some time" to give work for the job a chance to accumulate. Thats why I did the JavaDoc changes.
So albeit you wanted to specify this from a different mathematical point of reference, I hope you can at least agree that the new JavaDoc is just as correct as the old one?